themes/godotengine/pages/education.htm

```diff
@@ -0,0 +1,83 @@
+title = "Education"
+url = "/education"
+layout = "default"
+is_hidden = 0
+==
+{##}
+{% put head_text %}
+<p class="small">
+  Godot is already being used in many classrooms across the world. If you are interested in including it in your
+  curriculum, you can find answers to some of the most common questions on this page.
+</p>
+{% endput %}
+
+<style>
+  .question h3 {
+    margin: 2em 0 0;
+  }
+
+  .question p {
+    margin-top: 0.5em;
+  }
+</style>
+
+<div class="question">
+  <h3>Can I teach Godot in my classroom?</h3>
+  <p>Godot&nbsp;Engine is <a href="https://en.wikipedia.org/wiki/Free_and_open_source_software">free and open source
+      software</a> released under the permissive <a href="/license">MIT license</a>.<br>
+    You can use Godot in your classroom without any restrictions and with zero licensing
+    costs.
+  </p>
+  <p>You can read more about <a href="/license">our license here</a>.</p>
+</div>
+
+<div class="question">
+  <h3>Will the Godot project sign a services agreement provided by my institution?</h3>
+  <p>The Godot Engine project is unable to accept any terms and conditions outside of the MIT license, nor will it
+    provide warranties or representations about the Godot Engine outside those of the <a
+      href="/license">MIT license</a>.</p>
+  <p>As a free and open source project, Godot is not backed by a corporation or any individual. Accordingly, it doesn't
+    have the legal capacity to enter into agreements, even if such agreements did not conflict with the MIT license.</p>
+</div>
+
+<div class="question">
+  <h3>What data does the Godot Engine collect from users?</h3>
+  <p>When you use the Asset Library in the Godot Engine application, HTTPS requests are made to the Asset Library
+    website, and will contain: Godot Engine version, search query, selected assets, and the user's IP address. The Asset
+    Library
+    processes this data without storing it, but our TuxFamily hosting's HTTP server may log these requests. Such logs
+    are subject to <a href="https://faq.tuxfamily.org/Privacy/En" target="_blank">TuxFamily's privacy policy</a>.<br>
+    This is the only personal data collected by the Godot Engine application.</p>
+  <p>To be clear, the Godot Engine executable does not ask for or otherwise collect personally identifying information
+    from
+    its users. The network requests include the bare minimum amount of information required to do an online search and
+    transmit substantially less information than most internet browsing will.</p>
+    </p> For more information, please see <a href="/privacy-policy">our privacy policy here</a>
+</div>
+
+<div class="question">
+  <h3>I don't have permission to install software on my devices. How can I use Godot?</h3>
```

Comment:
Should this maybe mention self contained mode with `_sc_` file? I had some issues saving settings on a PC where I had no access to AppData (due to the user being on a network drive), so self contained mode might help some school environments... Can't find the docs for the feature right now, but those should then be linked too.

Reply:
Self-contained mode is documented on https://docs.godotengine.org/en/stable/tutorials/io/data_paths.html#self-contained-mode.

Reply:
Thats a great idea, but I would make a further PR for it as the exact details still need to be discussed and it is out of scope of Emilio's work so far